test-end-to-end/README.md

```diff
@@ -57,9 +55,9 @@ adding from device, test downloading from another device.
 4.1 Teacher can create assignment (set to allow one submission per student as per default), student 
     can submit assignment (text and attachment), teacher can view submission and grade it. After 
     submitting their work, the submit button is no longer visible. Student will see their mark when 
-    it is graded.
+    it is graded. Assignments can be edited, deleted. **Note: Users may expect to be able to resubmit an assigment provided it is within deadlines. ** 
 
-4.2 Teachers and students can add course comments which are visible for all who can view the assignment.
+4.2 Teachers and students can add course comments which are visible for all who can view the assignment. Course comments can be edited, deleted by owners/admin accounts
```

Comment:
Edit access by comment owners. Admins/teachers should be only be able to delete

Reply:
Edit by owner (e.g. the original poster, and it would need to be flagged as edited), and deletion by users who have permission to edit the course (e.g. teacher or admin) makes sense to me

It isn't part of any existing milestone. It wouldn't be a huge amount of additional work, and could be added to a milestone if the checklist is formally accepted.